.github/workflows/unroute-new-issue.yml

```diff
@@ -0,0 +1,33 @@
+name: Unroute new issue
```

Comment:
We don't even need this (if you are willing to let go of the org-member exception) as we can just do

```yaml
labels: ['Status: Unrouted']
```

inside the new templates and call it a day.

Reply:
Per https://github.com/getsentry/sentry/pull/26898#issuecomment-868826922, I think the org-member distinction is worth preserving.